packages/appsync-emulator-serverless/tester.js

```diff
@@ -3,13 +3,26 @@ const { createSchema } = require('./schemaTest');
 const createServerCore = require('./serverCore');
 const testJWT = require('./testJWT');
 const dynamoEmulator = require('@conduitvc/dynamodb-emulator/client');
+const { DynamoDB } = require('aws-sdk');
 
-const create = async ({ serverless, schemaPath, port = 0 } = {}) => {
-  // For performance we leverage a single emulator instance per process.
-  // To keep things unqiue between runs we use table names which are specific
-  // to each invocation of 'create'
-  const emulator = await dynamoEmulator.launch();
-  const dynamodb = dynamoEmulator.getClient(emulator);
+const create = async ({
+  serverless,
+  schemaPath,
+  port = 0,
+  dynamodbConfig = null,
+} = {}) => {
+  let dynamodb;
+  let emulator;
+
+  if (dynamodbConfig) {
+    dynamodb = new DynamoDB(dynamodbConfig);
```

Comment:
Nit: add quick comment about why we're not spinning up an emulator in this case.